### DISCUSSION TITLE
保全作业黑名单功能

### DISCUSSION THREAD

**Discussion by ZzzHe2333** [MaaAssistantArknights/MaaAssistantArknights#3917] [Ideas]
总所周知，保全必须选**满20个才能开始**。如果说，刚好不满足作业需求干员这个情况的话，他好像就会下的很奇怪。
【因为他玩我的号玩的很奇怪，我2022.10左右刚玩的，干员不多，他特别喜欢下星极，然后就漏，我不带还不够20个..】
看看能不能做到这样。
MAA新功能
·作业界面增加保全黑名单（设置成中介角色或者根本不用）。
·

（就这样吧）

**Comment by ABA2396** [comment]
保全的作业文件里有blacklist字段，里面的就是不会用的人
![image](https://user-images.githubusercontent.com/99072975/223686249-12a21eaf-effd-4d2d-ad4d-53947cae2f66.png)

**Reply by ZzzHe2333** [reply]
emmm我的意思是，不改文件，就直接MAA上修改的那种

**Reply by ZzzHe2333** [reply]
这样解释比较好：我目前没有这个角色或者这个角色很烂，我希望识别的时候，能临时把这个角色排除掉。等我后期角色起来了，我可以在MAA上直观的看出我Ban了什么角色，我可以在MAA上移除掉，免得打开文件的方法让自己遗忘。

**Reply by ZzzHe2333** [reply]
就可能新增一个黑名单.txt或者黑名单.json，跑程序的时候，根据任务是“保全”或者这个是“水月肉鸽”，自动屏蔽角色，并且表现在MAA程序上

**Reply by ZzzHe2333** [reply]
相当于更高级的大名单？可以对一系列生效？
